src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,46 @@
 'use strict';
-
 class Animal {
-  // write your code here
+  constructor(health, name, hidden) {
+    this.name = name;
+    this.hidden = hidden || false;
+    this.health = health || 100;
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(health, name, hidden) {
+    super(health, name, hidden);
+    this.name = name;
+    this.hidden = hidden || false;
+    this.health = 100;
```

Comment:
Those values will be setten in Animal constructor
```suggestion
    this.hidden = hidden || false;
```